docs/www/quick-start-cloud.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Vectorized Cloud Quick Start Guide
+order: 0
+---
+
+Whether for small projects or large-scale deployments, why manage your Redpanda infrastructure yourself when you can leave the heavy-lifting to us?
+
+Vectorized Cloud gives you cloud-based Redpanda services with security and stability without the need to manage your own clusters.
+Either you can choose a _fully-managed_ cluster and have the cluster stay in our cloud infrastructure,
+or you can use our _BYOC_ option and host the clusters in your cloud account.
+In both cases, our dedicated team of engineers monitor the cluster and take care of any maintenance issues that come up.
+
+After you set up the cluster, you'll get an endpoint and security credentials to allow your clients to access the cluster.
+You can use PandaProxy HTTP requests to manage your topics and partitions,
+but we also give you a dashboard with real-time metrics, basic topic management, and security controls for cluster and data access.
+
+## Sign in to Vectorized Cloud
+
+You can register for an account at (cloud.vectorized.io) with an email address and password or with a Google account.
+
+## Add a namespace
+
+Vectorized Cloud clusters are grouped into namespaces to help you organize your clusters in groups for convenience.
+
+Click **Add namespace** to create a namespace and give it a name.
+
+## Add a cluster
+
+In your namespace, click **Add cluster** and then choose which type of cluster to add:
+
+<tabs>
+
+  <tab id="Fully-managed">
+
+    Fully-managed clusters are hosted in the Vectorized Cloud account.
+
+    1. Enter a name for the cluster.
+    2. Select a cloud provider: **AWS** or **GCP**
+    3. Select a region in the provider.
+    4. Select a size for the node instances.
+    5. Enter the number of nodes for the cluster.
+    6. Click **Create** to initiate cluster creation.
+
+    After the cluster is created in Vectorized Cloud, you can click on the cluster in the namespace to see the cluster's connection details.
+
+  </tab>
+
+  <tab id="BYOC">
+
+    1. Enter a name for the cluster.
+    2. Select a cloud provider: **AWS** or **GCP**
+    3. Select a region in the provider.
+
+    After the cluster is created in Vectorized Cloud, you have to click **Setup cluster security** and follow the instructions to create the cluster in your cloud account.
+    The steps include:
+
+    1. Download and extract a zip archive of Terraform files.
```

Comment:
I think we should mention that terraform has to be preinstalled on the machine executing the commands.